roles/foreman_repositories/tasks/redhat.yml

```diff
@@ -48,8 +48,9 @@
       register: disabled_repos
       changed_when: false
 
-    - name: Enable powertools
-      command: dnf config-manager --set-enabled powertools
+    - name: Enable PowerTools
+      block:
+        - name: Enable PowerTools
+          command: dnf config-manager --set-enabled powertools
       when:
-        - ansible_distribution == 'CentOS'
-        - "'powertools' in disabled_repos.stdout"
+        - "'PowerTools' in disabled_repos.stdout"
```

Comment:
I think you want to check `powertools` not `PowerTools` here ;)

Reply:
Yeah, you're right :) both are on the same line, but the repo name is ```powertools```